bin/eslint-transforms.js

```diff
@@ -2,15 +2,44 @@
 
 "use strict";
 
-var nodeCLI = require("shelljs-nodecli");
+var execSync = require("child_process").execSync;
+var objectAssign = require("object-assign");
```

Comment:
Would it be okay to do that here already? I thought it would perhaps be better if I clean this up when I submit a separate PR for #4